backend/execution_cache/deployer/Dockerfile

```diff
@@ -0,0 +1,24 @@
+FROM alpine:3.8
```

Comment:
You can start from gcr.io/google.com/cloudsdktool/cloud-sdk or google/cloud-sdk so that you do not need to install the SDK.